addon/components/uni-tooltip.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import layout from '../templates/components/uni-tooltip';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  tagName: 'span',
+  classNames: ['uni-tooltip'],
+  classNameBindings: ['isOpen:uni-tooltip--active'],
+  layout,
+
+  informationIcon: false,
+  questionIcon: false,
+  isOpen: false,
+
+  click() {
+    this.set('isOpen', !this.get('isOpen'));
```

Comment:
I believe it is always useful to have a function that executes after the default "actions"

```js
onClick() {},

click() {
    this.set('isOpen', !this.get('isOpen'));

    this.get('onClick')();
}
```